samples/interactive-tutorials/product/import_products_big_query_table.js

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main() {
+  // [START retail_import_products_from_big_query]
+
+  // Imports the Google Cloud client library.
+  const {ProductServiceClient} = require('@google-cloud/retail').v2;
+
+  const projectNumber = process.env['PROJECT_NUMBER'];
+  const projectId = process.env['PROJECT_ID'];
```

Comment:
This variable hsould be `GCLOUD_PROJECT`, and will be set by default in the environment.